thrust/thrust/advance.h

```diff
@@ -30,8 +30,8 @@
 
 THRUST_NAMESPACE_BEGIN
 
-using ::cuda::std::advance;
-using ::cuda::std::next;
-using ::cuda::std::prev;
+using CCCL_DEPRECATED_BECAUSE("Use cuda::std::advance instead")::cuda::std::advance;
+using CCCL_DEPRECATED_BECAUSE("Use cuda::std::next instead")::cuda::std::next;
+using CCCL_DEPRECATED_BECAUSE("Use cuda::std::prev instead")::cuda::std::prev;
```

Comment:
Please add comments, that we deprecated those entities in CCCL 3.1, so we know they have to stay deprecated for a while:
```suggestion
//! deprecated [since 3.1]
using CCCL_DEPRECATED_BECAUSE("Use cuda::std::advance instead")::cuda::std::advance;
//! deprecated [since 3.1]
using CCCL_DEPRECATED_BECAUSE("Use cuda::std::next instead")::cuda::std::next;
//! deprecated [since 3.1]
using CCCL_DEPRECATED_BECAUSE("Use cuda::std::prev instead")::cuda::std::prev;
```